src/go/k8s/controllers/redpanda/cluster_controller_configuration.go

```diff
@@ -401,7 +401,7 @@ func needsRestart(
 ) bool {
 	nodeNeedsRestart := false
 	for i := range clusterStatus {
-		log.Info(fmt.Sprintf("Node %d restart status is %v", clusterStatus[i].NodeID, clusterStatus[i].Restart))
+		log.Info(fmt.Sprintf("Node %d restart status is %v", clusterStatus[i].NodeID, clusterStatus[i].Restart)) // actually pod ordinal
```

Comment:
I still see the comment. Should it be removed as https://github.com/redpanda-data/redpanda/pull/7581#discussion_r1049962886